datadog/dogstatsd/base.py

```diff
@@ -29,13 +29,30 @@
 DEFAULT_HOST = 'localhost'
 DEFAULT_PORT = 8125
 
-# Tag name of entity_id
-ENTITY_ID_TAG_NAME = "dd.internal.entity_id"
+# Mapping of DD_* env vars and the tag names that should be used for
+# setting their values. Note that `DD_TAGS` is treated separately because
+# its value can contain _multiple_ tags.
+DD_ENV_TAGS_MAPPING = {
+    'DD_ENTITY_ID': 'dd.internal.entity_id',
+    'DD_ENV': 'env',
+    'DD_SERVICE': 'service',
+    'DD_VERSION': 'version',
+}
 
 # Telemetry minimum flush interval in seconds
 DEFAULT_TELEMETRY_MIN_FLUSH_INTERVAL = 10
 
 
+def get_dd_tags():
```

Comment:
can we move it to utils and/or prefix it with _ in case it is not expected to be a public API?